source/docs/advanced/production-ready/overview.md

```diff
@@ -8,6 +8,7 @@ In this section, you will learn advanced information about the different things
 - [Optimize your media](./media-middleware.html)
 - [Configure multiple stores](./multistore.html)
 - [Setup and use logging](./logging.html)
+- [Setup sessions](./sessions.html)
```

Comment:
**issue:** the page is not available in the navigation menu

**suggestion:** add it in the config.yml menu list too

Reply:
added